### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by ping035627**
What type of PR is this?
/kind cleanup

**What this PR does / why we need it: **
fix some typos.

Which issue(s) this PR fixes (optional, in fixes #(, fixes #<issue_number>, ...) format, will close the issue(s) when PR gets merged):
Fixes #

Special notes for your reviewer:

Does this PR introduce a user-facing change?:

/release-note-none
/priority backlog

### CODE REVIEW

File: pkg/kubelet/cm/devicemanager/manager.go [hunk 1]
```diff
@@ -669,7 +669,7 @@ func (m *ManagerImpl) allocateContainerResources(pod *v1.Pod, container *v1.Cont
 		// and container2 both require X resource A and Y resource B. Both allocation
 		// requests may fail if we serve them in mixed order.
 		// TODO: may revisit this part later if we see inefficient resource allocation
-		// in real use as the result of this. Should also consider to parallize device
+		// in real use as the result of this. Should also consider to paralyze device
```
Comment (RenaudWasTaken) [inline pkg/kubelet/cm/devicemanager/manager.go]: typo here, should be parallelize

Comment (ping035627) [reply pkg/kubelet/cm/devicemanager/manager.go]: oh, yes, I have corrected it, thanks for reminder.

**Review by RenaudWasTaken — COMMENTED** [review]
/lgtm

@jiayingz for approval

**Comment by ping035627** [comment]
@jiayingz can you review this pr? Need approved label. Thanks.

**Comment by ping035627** [comment]
@vishh @jiayingz can you review this pr? Need approved label. Thanks.

**Comment by ping035627** [comment]
Why the pull-kubernetes-godeps did not report status?

**Pull request merged**